Source/WebCore/platform/mediastream/gstreamer/RealtimeIncomingSourceGStreamer.cpp

```diff
@@ -79,6 +97,20 @@ void RealtimeIncomingSourceGStreamer::registerClient()
         return GST_FLOW_OK;
     }), this);
 
+    g_signal_connect_swapped(sink, "new-serialized-event", G_CALLBACK(+[](RealtimeIncomingSourceGStreamer* self, GstElement* sink) -> gboolean {
+        auto object = adoptGRef(gst_app_sink_pull_object(GST_APP_SINK(sink)));
+        auto event = GST_EVENT_CAST(object.get());
```

Comment:
I think you can do something like: `auto event = adoptGRef(GST_EVENT_CAST(gst_app_sink_pull_object(GST_APP_SINK(sink))))` and then `WTFMove` below.

Reply:
Nice, thanks!